getting_started/workflow/export/exporting_for_ios.rst

```diff
@@ -3,70 +3,87 @@
 
-Loading files from a host
--------------------------
+  * **Path** should be an empty folder that will contain your exported Xcode project files. 
+  * **File** will be the name of your Xcode project and several project specific files and directories.  
 
-Sometimes your game becomes too big and deploying to the device takes
-too long every time you run. In that case you can deploy only the engine
-executable, and serve the game files from your computer.
+.. image:: img/ios_export_file.png
 
-Setting up the file host
-~~~~~~~~~~~~~~~~~~~~~~~~
+When the export completes, the options I used will create a directory that looks like this:
 
-On your PC, open the editor, and click the righ-most icon on the
-top-center group of icons, and select "Enable File Server". The icon
-turns red. Your PC will open a port and accept connections to serve
-files from your project's directory (so enable your local firewall
-accordingly).
+.. image:: img/ios_export_output.png
 
-.. image:: img/rfs_server.png
+I can now open **exported_xcode_project_name.xcodeproj** to build and deploy my game
+like any other iOS app. 
 
-Setting up the game
-~~~~~~~~~~~~~~~~~~~
 
-On XCode, click on your app name (top left, next to the "Stop" button),
-and select "Edit Scheme". Go to the "Arguments" tab, and add 2
-arguments, "-rfs" and the IP of your PC.
+Active development considerations
+---------------------------------
 
-.. image:: img/edit_scheme.png
+The above method creates an exported project that you can build for release, but you
+have to re-export every time you make a change in Godot. 
+
+While developing your game, you can speed this process up by building your Godot project
+files directly into your app. 
+
+In the following example:
+
+  * **exported_xcode_project_name** is the name of the exported iOS application (as above). 
+  * **godot_project_to_export** is the name of the Godot project I exported it from. 
+
+Steps to use your Godot project directory in Xcode
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Start from an exported iOS project like I created above.
+2. In Finder, drag your Godot project folder into the Xcode file browser. 
+
+.. image:: img/ios_export_add_dir.png
+
+3. In the dialog, make sure **Create folder references** is selected. This means
+that you can continue to edit your Godot project in its current location.
+
+.. image:: img/ios_export_file_ref.png
+
+4. You now see the **godot_project_to_export** directory in the Xcode file browser. 
+5. Delete **exported_xcode_project_name.pck** from the Xcode project.
+
+.. image:: img/ios_export_delete_pck.png
+
+6. Open **exported_xcode_project_name-Info.plist** and add a string property named
+**godot_path** (this is the real key name) with a value **godot_project_to_export** 
+(this is the name of your project)
+
+.. image:: img/ios_export_set_path.png
+
+That's it! Continue to edit your project in the Godot editor and build it in Xcode when you 
+want to run it on a device. 
```

Comment:
Nice, I wasn't even aware this works. I guess we could add an export option that would create these references instead of exporting .pck.

Reply:
Yeah, it was a nice surprise. 

I'm still trying to figure out the RFS option, but this is a pretty huge win in terms of development efficiency.